R/AnalyzeUsingHazardRatioLimitsOfCI.R

```diff
@@ -0,0 +1,138 @@
+#' @param AnalyzeUsingHazardRatioLimitsOfCI
+#' @title Analyze using a simplified limits of confidence interval design
+#' @param SimData Data frame which consists of data generated in current simulation.
+#' @param DesignParam List of Design and Simulation Parameters required to perform analysis.
+#' @param LookInfo List containing Design and Simulation Parameters, which might be required to perform analysis.
+#' @param UserParam A list of user defined parameters in East or Solara. UserParam must be supplied, the list must contain the following named elements:
+#' \describe{
+#'   \item{UserParam$dMAV}{A value (0, Inf) that specifics the lower limit, eg  Minimum Acceptable Value (MAV). }
+#'   \item{UserParam$dTV}{A value (0 Inf) that specifies the upper limit for the confidence interval, eg Target Value (TV).}
+#'   \item{UserParam$dConfLevel}{A value (0,1) that specifies the confidence level for the t.test() function in base R library.}
+#' }
+#' @description  In this simplified example of upper and lower confidence boundary designs, if it is likely that the HR (Hazard Ratio) is below the Minimum Acceptable Value (MAV) then a Go decision is made.  
+#'               If a Go decision is not made, then if it is unlikely that the Hazard ratio is below the Target Value (TV) a No Go decision is made.      
+#'               In this example, the coxph() from survival package in R is utilized to analyze the data and compute estimate of log HR and Std error of log HR. 
+#'               The team would like to make a Go decision if there is at least a 90% chance that HR is below than the MAV.  
+#'               If a Go decision is not made, then a No Go decision is made if there is less than a 10% chance the difference is less than the TV.  
+#'          
+#'               Specifically, if user provides upper and lower limit in Hazard ratio scale then,
+#'               1. For Hazard Ratio 

```

Comment:
A lower Hazard Ratio is a good thing.   So, in this setting the rule is generally something like:
1. If UL < MAV --> Go
2. If LL > TV --> No Go
Otherwise go to the next Analysis
Typically at the FA it is something like if a Go is not made then it is a No Go.  

Here is a picture of this type, the Yellow consider is only at the IA, typically that becomes a No Go at the end.
![image](https://github.com/Cytel-Inc/CyneRgy/assets/32512561/4e71da0f-2281-4547-b721-18554120ca11)

Reply:
Noted.